container/build.sh

```diff
@@ -455,13 +466,14 @@ show_help() {
     echo "  [--dry-run print docker commands without running]"
     echo "  [--build-context name=path to add build context]"
     echo "  [--release-build perform a release build]"
-    echo "  [--make-efa Enables EFA support for NIXL]"
+    echo "  [--make-efa Adds AWS EFA layer on top of the built image (works with any target)]"
```

Comment:
nit:  Non blocking - but 'enable' or 'add' seems a little more intuitive - but again not a blocker

```suggestion
    echo "  [--enable-efa Adds AWS EFA layer on top of the built image (works with any target)]"
```